crates/starknet-devnet-server/test_data/spec/0.8.0-rc3/starknet_ws_api.json

```diff
@@ -355,11 +385,7 @@
       "TOO_MANY_BLOCKS_BACK": {
         "code": 68,
         "message": "Cannot go back more than 1024 blocks"
-      },
-      "CALL_ON_PENDING": {
```

Comment:
Should this also be addressed in the code? There is a `ApiError::CallOnPending` entry in our errors. Or did you just imagine this PR as a spec file update?